build/upgrade_utils.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+bump_manifest_version() {
+    echo "-----------------------------------------------------------------------------------"
+    echo "Bumping $REPO_SRC_DIR version in manifest.json"
+    echo "-----------------------------------------------------------------------------------"
+    VERSION=$(cd $REPO_SRC_DIR && ../build/current_version.sh) || return 1
```

Comment:
Should these values be passed in as arguments rather than set as shared variables? Do these helpers provide enough utility to live separately and justify the extra cognitive overhead?